packages/valory/skills/decision_maker_abci/behaviours/decision_receive.py

```diff
@@ -164,6 +164,16 @@ def _get_response(self) -> None:
 
         self._mech_response = mech_responses[0]
 
+    def _get_weighted_accuracy(self) -> Optional[float]:
+        tool_name = self.synchronized_data.mech_tool
+        store_tools = list(self.policy.weighted_accuracy.keys())
```

Comment:
The `_policy` attribute is not set anywhere. This is going to raise.

Reply:
moved this logic to the Synchronized data